### ISSUE TITLE
Latest version to download is not the same as the latest release on github

### ISSUE THREAD

**Issue by kachkaev** [duplicati/duplicati.github.io#10]
At the moment, https://www.duplicati.com/download suggests downloading `2.0.3.3_beta_2018-04-02` (see https://updates.duplicati.com/beta/latest-installers.js). However, the [releases page](https://github.com/duplicati/duplicati/releases) mentions a few new versions up to `2.0.3.11_canary_2018-09-05`.

I'm new to duplicati and would like to install it on macOS via homebrew cask. The [current formula](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/duplicati.rb) refers to `2.0.3.3,2018-04-02`, just like the webpage. When @mik-at wanted to bump the version to `2.0.3.5,2018-04-13`, [the PR](https://github.com/Homebrew/homebrew-cask/pull/45946) got rejected to maintain the match between the version on the website and the version in the cask.

What is the reason for suggesting people to download the April version given that more recent betas are available? When are you planning to bump the version at https://www.duplicati.com/download? I'm happy to submit a new PR to brew cask once this is done and start exploring the very latest version of duplicati 🚀 

---
сс @Pectojin (as someone who is not subscribed to this repo, but has merged quite a few duplicati PRs recently)

**Comment by Pectojin** [comment]
`v2.0.3.3` is the latest beta build. Kenneth did a lot of work on multi threaded processing afterwards and we're still working out the kinks in canary. 

The beta is by far recommended for most people. Partly because of the multi threading and partly because very few people have tested the code going into Canary.

I don't manage the homebrew repo, but I'd suggest putting the canary in a duplicati-canary item. There are tons of other brews offering `name-beta` or `name-canary` versions such as Discord, Virtualbox, and Google Chrome, so there is definitely precedent.
